src/LibraryManager/Manifest.cs

```diff
@@ -99,8 +111,7 @@ public static Manifest FromJson(string json, IDependencies dependencies)
                 Manifest manifest = JsonConvert.DeserializeObject<Manifest>(json);
                 manifest._dependencies = dependencies;
                 manifest._hostInteraction = dependencies.GetHostInteractions();
-
-                UpdateLibraryProviderAndDestination(manifest);
+                manifest._librariesValidator = new LibrariesValidator(dependencies, manifest.DefaultDestination, manifest.DefaultProvider);
```

Comment:
> new LibrariesValidator [](start = 46, length = 23)

This is going to make unit testing hard if we ever want to stub out the LibrariesValidator.